app/src/main/res/layout-sw600dp-port/admin_pin_activity.xml

```diff
@@ -65,35 +64,53 @@
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_pin"
+        <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/admin_pin_input_pin"
+          style="@style/TextInputLayout"
           android:layout_width="0dp"
           android:layout_height="wrap_content"
           android:layout_marginStart="28dp"
           android:layout_marginTop="28dp"
           android:layout_marginEnd="28dp"
-          app:inputLength="5"
-          app:isPasswordInput="true"
-          app:label="@string/admin_pin_new_pin"
+          app:errorMessage="@{viewModel.pinErrorMsg}"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text"
-          profile:error="@{viewModel.pinErrorMsg}" />
+          app:layout_constraintTop_toBottomOf="@+id/admin_pin_warning_text">
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_confirm_pin"
+          <com.google.android.material.textfield.TextInputEditText
```

Comment:
Add this to support RTL for TextInputLayout
`android:textDirection="locale"`

Reply:
done